examples/lit-icd-app/lit-icd-common/lit-icd-server-app.matter

```diff
@@ -1739,8 +1739,8 @@ endpoint 0 {
     callback attribute registeredClients;
     callback attribute ICDCounter;
     callback attribute clientsSupportedPerFabric;
-    callback attribute userActiveModeTriggerHint;
-    callback attribute userActiveModeTriggerInstruction;
+    ram      attribute userActiveModeTriggerHint default = 0x110D;
+    persist  attribute userActiveModeTriggerInstruction default = "Restart the application";
```

Comment:
This is read-only. It should probably be ram instead.

Reply:
good catch! fixed